lib/types/relationship-types.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+export const knowOfFriendsStatus = Object.freeze({
+  KNOW_OF: 0,
+  FRIEND: 2,
+});
+export type KnowOfFriendStatus = $Values<typeof knowOfFriendsStatus>;
+
+export const requestsBlocksStatus = Object.freeze({
+  PENDING_FRIEND: 1,
+  BLOCKED: 3,
+});
+export type RequestsBlocksStatus = $Values<typeof requestsBlocksStatus>;
+
+type RelationshipAction =
+  | 'friend_request'
+  | 'accept_friend_request'
+  | 'cancel_friend_request'
+  | 'unfriend'
+  | 'block'
+  | 'unblock';
+export const relationshipActions = [
+  'friend_request',
+  'accept_friend_request',
+  'cancel_friend_request',
+  'unfriend',
+  'block',
+  'unblock',
+];
```

Comment:
I agree and disagree. I agree that it's better because everything follows from a single object. I disagree because `RelationshipAction` is typed as a 'string' instead of a union type so it's possible to pass every string to a function that accepts `RelationshipAction` enum so Flow won't catch it

Reply:
> it's possible to pass every string to a function that accepts `RelationshipAction` enum

I don't think this is correct. Have you tested it?

Reply:
Ok, you're right. Looks like I can no longer rely on what VSCode shows :(